### DISCUSSION TITLE
Can I specify any possible forecast interval in the plot function of the forecast object?

### DISCUSSION THREAD

**Discussion by MarleneBs** [awslabs/gluonts#2437] [Q&A] [answered]
Can I specify any possible forecast interval in the plot function of the forecast object?
So regardless of which quantiles I set in the evaluator?

**Comment by codingWhale13** [comment, marked as answer]
Yes, those are separate.
Note that the [evaluator](https://github.com/awslabs/gluonts/blob/0e4a488946fbdd8bf5a4da73997f399b575fda50/src/gluonts/evaluation/_base.py#L111-L113) expects the quantile levels to be floats between 0 and 1 while the [plot function](https://github.com/awslabs/gluonts/blob/0e4a488946fbdd8bf5a4da73997f399b575fda50/src/gluonts/model/forecast.py#L357-L360) expects percentages, so values between 0 and 100.

**Reply by MarleneBs** [reply]
Thank you very much :)
